### PR TITLE
Mec view user posts

### PR DESCRIPTION
**Pull request by mecrowley**
#### Changes Made

1. Added file `request.py` to `posts` directory.
2. Added file 'post.py' to 'models' directory.
3. Added file '__init__.py' and 'request.py' to 'post_reactions' directory.
4. Added file 'post_reaction' to 'models' directory.
5. Modified file `request_handler.py` to include `get_single_user` in do_GET()
6. Modified file `request_handler.py' to include 'get_posts_by_user' in do_GET()
7. Modified file 'request_handler.py' to include 'get_all_post_reactions' in do_GET()
8. Modified file 'rare-init.sql' to include INSERT INTO SQL queries to seed Posts and PostReactions Tables

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout mec-view-user-posts
   ```
2. Ensure the backend and client are both running.
4. Test app functionality.

   > When user runs GET http://localhost:8088/users/{id} in Postman, the user object with that id should be returned
   > When user runs GET http://localhost:8088/posts?user_id={id}, all the post objects with that user_id value should be returned
   > When user runs GET http://localhost:8088/postreactions, all the post reactions should be returned

5. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.

### CODE REVIEW

**Review by alambrow — APPROVED** [review]
Ran all of the fetch calls, and all return the proper data!

File: .pylintrc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[FORMAT]
+good-names=i,
+    j,
```
Comment (enbeec) [inline .pylintrc]: :question:

**Review by enbeec — APPROVED** [review]
Looks good! There's some stuff in the dictionary that doesn't appear in the codebase. Probably fine just wanted to flag it :smile:

File: .pylintrc (hunk 1 above)
Comment (mecrowley) [reply .pylintrc]: bonus code

**Pull request merged**